data/auxiliary/gather/query_ldap/default.yaml

```diff
@@ -0,0 +1,98 @@
+---
+queries:
+  - action: ENUM_ALL_OBJECT_CLASS
+    description: 'Dump all objects containing any objectClass field.'
+    filter: '(objectClass=*)'
+    columns:
```

Comment:
```suggestion
    attributes:
```

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/16598/commits/cca391757323058c5b6f8fcebe1e178ce21d3f97, good point on this change :)